### DISCUSSION TITLE
add my work

### DISCUSSION THREAD

**Discussion by Fr-Dae** [Rycieos/factorio-colorblind-ultimate#6] [Ideas]
Hello, i'm the maintener of colorblind mod
thanks you for your mod and your work
could you add some of my modification at yours ? i have essencially modified the color of icon (like adv circuit or cable) 

https://mods.factorio.com/mod/Colorblind_AllInOne

it's opensources, just mention Fr_Dae on credit, no need more =)

**Comment by Rycieos** [comment]
I'm glad you are finding the mod useful! And welcome!

I am always happy to add functionality to address any visual issues that anyone has playing the game. That said, I am not sure how adding some of your custom icons would achieve that goal. Firstly, my mod is designed to be useful for any form of color blindness, even black and white vision. The icons in your mod are simply color shifted, meaning they would help some people but not others. Unless there is no other option (for example, the range indicators that the game engine can only use a solid color for), my goal is to use shape or size to differentiate, never just color.

Second, most of the graphics in your mod are for items or entities that my mod already supports. If there are specific objects that are not supported that you would like to see supported, please mention them. If there are objects that are supported that you find that none of the available options work for you, please let me know.

Lastly, if you have custom icons that include more changes than simply color, I would be happy to include them in this mod. For examples, see the upper half of the [mod's thumbnail](https://github.com/Rycieos/factorio-colorblind-ultimate/blob/03819541796afe5c11721100fa50ac8a2aacf5c2/thumbnail.png).

**Comment by Fr-Dae** [comment]
I've focused on green and red perception problems, which are the most common, and I've changed the icons of certain objects, especially the circuits, 
because once they're stacked on a carpet it's hard to tell them apart from anything else, and the game's colour-blind mode doesn't really help.

**Reply by Rycieos** [reply]
You didn't address the contents of [my message above](https://github.com/Rycieos/factorio-colorblind-ultimate/discussions/6#discussioncomment-11199482).

> I've focused on green and red perception problems, which are the most common

I agree they are the most common, but my mod is designed to be used by all forms of colorblindness.

> and I've changed the icons of certain objects, especially the circuits

My mod does the same.

> and the game's colour-blind mode doesn't really help.

I agree, that is why I made my mod.

I do not see an argument for _why_ I should include any of your icons or sprites. If you want me to include something, answer these questions (for each icon/sprite):
1. What should be included (ex: electronic circuit custom icon)
2. Why your icon is better than mine, or why mine isn't good enough and yours should be a second option.

**Reply by Fr-Dae** [reply]
it's not better, it's just purple =) adv chip especial llok cool in purple

**Reply by Rycieos** [reply]
Well unfortunately, making the advanced circuit purple makes it much harder for me to tell the difference between advanced circuits and processing units. And I likely am not the only one. I also prefer [my custom icon](https://github.com/Rycieos/factorio-colorblind-ultimate/blob/main/graphics/icons/advanced-circuit.png) anyway.

If you have _specific_ suggestions that do not include simply color shifting, please add them. Otherwise, I am closing this discussion as the mentioned mod only contains color shifted graphics.
